cmd/gen-go-sample/inittree.go

```diff
@@ -171,11 +171,11 @@ func (t *initTree) parseSampleArgPath(txt string, info pbinfo.Info, varName stri
 	return &cp, report(nil)
 }
 
-// parsePath parses a path as defined below. It returns the subtree specified by the path,
+// parsePathRest parses a path as defined below. It returns the subtree specified by the path,
 // the last scanned token, and any error.
 //
-// path = ident { '.' ident } .
-func (t *initTree) parsePath(sc *scanner.Scanner, info pbinfo.Info) (*initTree, rune, error) {
+// path = { '.' ident } .
+func (t *initTree) parsePathRest(sc *scanner.Scanner, info pbinfo.Info) (*initTree, rune, error) {
```

Comment:
Why did you change the name? What does "Rest" mean here?

Reply:
Slightly edited the comment. The "rest" essentially means this function parses the "rest" of the path.

The full path is, for example, "a.b.c". This function expects that "a" has already been parsed by the caller and parses ".b.c" section.